ams/ams-server/src/main/java/com/netease/arctic/ams/server/utils/CatalogUtil.java

```diff
@@ -40,6 +42,8 @@
  * catalog util class。cache thrift objects
  */
 public class CatalogUtil {
+
+  private static final Logger LOG = LoggerFactory.getLogger(CatalogUtil.class);
   public static final ConcurrentHashMap<String, ArcticCatalog> catalogCache = new ConcurrentHashMap<>();
```

Comment:
Nit: `catalogCache` should be `CATALOG_CACHE`

Reply:
done